cognite/client/data_classes/data_modeling/instances.py

```diff
@@ -1011,12 +1021,38 @@ def as_write(self) -> NodeApplyList:
         """Returns this NodeList as a NodeApplyList"""
         return NodeApplyList([node.as_write() for node in self])
 
+    @classmethod
+    def _load_raw_api_response(cls, responses: list[dict[str, Any]], cognite_client: CogniteClient) -> Self:
+        typing: TypeInformation | None = None
+        if len(responses) >= 1:
+            typing = TypeInformation._load(responses[0]["typing"]) if "typing" in responses[0] else None
+        resources = [
+            cls._RESOURCE._load(item, cognite_client=cognite_client)  # type: ignore[has-type]
+            for response in responses
+            for item in response.get("items", [])
+        ]  # type: ignore[has-type]
+        return cls(resources, typing, cognite_client=cognite_client)
+
+    def dump(self, camel_case: bool = True) -> list[dict[str, Any]] | dict[str, Any]:  # type: ignore[override]
+        items = super().dump(camel_case)
+        if self.typing:
+            return {
+                "items": items,
+                "typing": self.typing.dump(camel_case),
+            }
+        else:
+            return items
```

Comment:
Just a thought, would it be better to have a separate "dump raw" method?

Reply:
Yes I agree, that is better.